### DISCUSSION TITLE
milestone

### DISCUSSION THREAD

**Discussion by djabmal** [EdgeTX/edgetx#1962] [General]
hello  why the milestone page is not updated ?it is stuck at 93% since a week although edge 7.1 is out already
https://github.com/EdgeTX/edgetx/milestones

**Comment by pfeerick** [comment]
The milestone didn't update because #1745 hasn't been merged normally, and the PR is still open.

**Comment by eshifri** [comment]
This PR is not relevant in 2.8, is it?

**Reply by pfeerick** [reply]
No, not anymore, since LVGL would have blew away anything this did. So that PR remains a 2.7.1 only fix, and can now be closed.

**Reply by raphaelcoeffic** [reply]
> No, not anymore, since LVGL would have blew away anything this did. So that PR remains a 2.7.1 only fix, and can now be closed.

I believe the new screens (those using flex layout) don't suffer from this issue. We still have to convert the old screens to the new *in-place modifications* patterns, so that the focus being lost should be a thing of the past anyway on `main`.
